packages/manager/src/managers/customTypes/CustomTypesManager.ts

```diff
@@ -82,6 +88,21 @@ type CustomTypesMachineManagerDeleteCustomTypeReturnType = {
 	errors: (DecodeError | HookError)[];
 };
 
+type CustomTypeFieldIdChangedMeta = NonNullable<
+	NonNullable<CustomTypeUpdateHookData["updateMeta"]>["fieldIdChanged"]
+>;
+
+type CrCustomTypes = readonly CrCustomType[];
```

Comment:
❓I think we should have that on types-internal no? If we change types-internal this needs to be aligned and I don't like that we just recreate it manually.

Can you track this to have it there if yes

Reply:
I agree, I was gonna suggest exporting this from types internal, but by removing the recursive there's less need for these types.

Reply:
So I propose you keep that in mind and if at the end of the recursive PR we still need that, we create a ticket for types internal, good for you?